src/Compilers/CSharp/Test/Symbol/Symbols/ModuleInitializersTests.Ignored.cs

```diff
@@ -0,0 +1,247 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Symbols
+{
+    public sealed partial class ModuleInitializersTests
+    {
+        [Fact]
+        public void IgnoredOnReturnValue()
+        {
+            string source = @"
+using System.Runtime.CompilerServices;
+
+class C
+{
+    [return: ModuleInitializer]
+    internal static void M()
+    {
+    }
+}
+
+namespace System.Runtime.CompilerServices { class ModuleInitializerAttribute : System.Attribute { } }
+";
+            CompileAndVerify(source, parseOptions: s_parseOptions, symbolValidator: module =>
+            {
+                var rootModuleType = (TypeSymbol)module.GlobalNamespace.GetMember("<Module>");
+                Assert.Null(rootModuleType.GetMember(".cctor"));
```

Comment:
>Assert.Null(rootModuleType.GetMember(".cctor")); [](start = 16, length = 48)

Since this method is private, we cannot trust this check unless ```MetadataImportOptions.All``` is used (you already did that in previous PR), because we might simply not import it. That option is probably not used by default, but I could be wrong. Please add an assert for ```ImportOptions``` on the module and do that for all similar tests to make them more robust.